### ISSUE TITLE
Create a Blacklight deployment

### ISSUE THREAD

**Issue by steveellis** [culibraries/blacklight-skunkworks#5]
There has been growing support for this idea so I'm going to go ahead and do it. I think the steps will be something like.

1. Get my AWS id from Mark
2. Figure out how to swap out sqlite for postgres
3. Containerize the code we have so far
4. Determine where things like the solr and db connection are in the code
5. Figure out how to get the db and solr out on RDS or wherever we're hosting those
6. Deploy the app container to AWS setting the various env vars there

Then once we have a running deployment, I'll study the various parts and figure out how to implement as a AWS pipeline. The goal should be that a commit on a given branch will trigger a new deployment.

Finally once all of this is done, I'll import some data from the catalog.

**Comment by steveellis** [comment]
This has gone pretty smoothly. Using @mbstacy's code in the ansible project as my starting point I was able to get a k8s deployment going in just a few days. The Dockerfile is what gets deployed for the app layer. Updated the readme with instructions for building and deploying the container. The app is now available at https://discovery.cublcta.com.

I was able to customize the look-and-feel easily to use the official CU boulder colors (obtained from the CU communications repo) with the bootstrap variables in the local.scss file.
